counted.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 RTrade Technologies Ltd
+//
+// licensed under GNU AFFERO GENERAL PUBLIC LICENSE;
+// you may not use this file except in compliance with the License;
+// You may obtain the license via the LICENSE file in the repository root;
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+package blockstore
```

Comment:
I think a blank line is needed to separate package description and copyright header in a go file.

Reply:
https://medium.com/@elliotchance/godoc-tips-tricks-cda6571549b
> Package level documentation must be in a continuous comment immediately preceding the packagestatement. If there is a blank line between the comment and the packagestatement it will be discarded. This is how you add comments that should or should not be included in the general package documentation.

Reply:
good catch, i have fixed this